### PR TITLE
Bug 1610709 -- new note about nodeSelector

### PR DESCRIPTION
**Pull request by jhoyt-rh**
@vikram-redhat  PTAL
@jsafrane  PTAL

No QE listed, since it doesn't apply here.

Bug is here:

https://bugzilla.redhat.com/show_bug.cgi?id=1610709

### CODE REVIEW

File: install_config/persistent_storage/persistent_storage_csi.adoc [hunk 1]
```diff
@@ -328,6 +328,8 @@ spec:
 <1> Replace with `cloud.conf` for your OpenStack deployment, as described in
  xref:../../install_config/configuring_openstack.adoc#configuring-openstack-variables[OpenStack configuration]. For example, the Secret can be generated using the `oc create secret generic cloudconfig --from-file cloud.conf --dry-run -o yaml`.
 
+<2> Optionally, add `nodeSelector` to the CSI driver pod template to configure the nodes on which the CSI driver should be started. Only nodes matching the selector will be able to run pods that use volumes served by the CSI driver. Without `nodeSelector`, the driver will run on all nodes in the cluster.
```
Comment (vikram-redhat) [inline install_config/persistent_storage/persistent_storage_csi.adoc]: @jhoyt-rh - Hi Joan - this is missing: https://github.com/openshift/openshift-docs/pull/11606/files#diff-fa532ad4b01ebfd61474800bc6c0c948L256.

The <2> is like a footnote that is referencing content that has changed earlier in the book. See Jan's original PR.

Comment (kalexand-rh) [reply install_config/persistent_storage/persistent_storage_csi.adoc]: Please edit the text to remove "should" and the future tense.
s/volumes served/volumes that are served

**Review by kalexand-rh — COMMENTED** [review]
Couple of picks

**Comment by jhoyt-rh** [comment]
Thanks, Kathryn.  Made your edit.

Joan Hoyt
Sr. Technical Writer, Customer Content Services/OpenShift
Red Hat, Inc
Westford, MA
jhoyt@redhat.com
857-770-0645

**Pull request merged**